vllm/attention/backends/torch_sdpa.py

```diff
@@ -8,8 +8,16 @@
 
 from vllm.attention.backends.abstract import (AttentionBackend, AttentionImpl,
                                               AttentionMetadata)
-from vllm.attention.ops.paged_attn import (PagedAttention,
-                                           PagedAttentionMetadata)
+from vllm.attention.ops.paged_attn import PagedAttentionMetadata
+from vllm.utils import is_cpu
+
+if is_cpu():
+    try:
+        from vllm.attention.ops.ipex_attn import PagedAttention
+    except ImportError:
+        from vllm.attention.ops.paged_attn import PagedAttention
```

Comment:
Can't we simply require users to use IPEX? In which case do we have to use the PagedAttention kernel in vLLM?

Reply:
Yes, after the APIs in IPEX become stable we will add IPEX to the requirements so the users can use it directly. We want to leave the native kernel here to evaluate some latest features (e.g., 8bit KV cache) before the IPEX supports them and public release.

Reply:
I see. Thanks!